### ISSUE TITLE
ML: VGG16 Model #1 (24 Layers): Configuration#4: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#268]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
50 epochs

Epoch 00014: val_accuracy did not improve from 0.87539

840000/840000 [==============================] - 336s 400us/sample - loss: 3.0591 - accuracy: 0.9036 - val_loss: 3.3555 - val_accuracy: 0.6629
Epoch 00014: early stopping

**Comment by hmehryar** [comment]
port to excel for overleaf
